<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_quick</class>
 <widget class="QDialog" name="add_quick">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Quick Reminder</string>
  </property>
  <property name="windowIcon">
   <iconset theme="remindor-qt">
    <normalon>:/icons/remindor-qt.svg</normalon>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="in_spin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_label">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QLabel" name="info_label">
     <property name="text">
      <string>Popup:
Dialog:
Sound file:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="in_slider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="sliderPosition">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="in_label2">
     <property name="text">
      <string>In</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="in_label">
     <property name="text">
      <string>In</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="minutes_label">
     <property name="text">
      <string>minutes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QWidget" name="button_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalspacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_button">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset theme="cancel">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_button">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset theme="add">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QLineEdit" name="label_edit">
     <property name="text">
      <string>Quick Reminder</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLabel" name="minutes_label2">
     <property name="text">
      <string>minutes</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="value_label">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string> 30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>in_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>in_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>in_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>in_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>in_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
